sig-windows/CONTRIBUTING.md

```diff
@@ -52,30 +53,36 @@ If you would like to build all binaries at once, then run `./build/run.sh make c
 
 Once the build completes, the files will be in `_output/dockerized/bin`.
 
-### Updating the Node binaries
+### Testing your changes
 
-Once you have binaries built, the easiest way to test them is to replace them on an existing cluster. This section assumes you already have a cluster in the cloud of your choice. To update the binaries on an existing node, follow these steps:
+#### Build a local cluster
 
-1. Drain & cordon a node with `kubectl drain <nodename>`
+The easist way to test changes is to use the community maintained [SIG Windows developer environment](https://github.com/kubernetes-sigs/sig-windows-dev-tools#welcome-to-the-sig-windows-development-environment) project.
+You can use this to build a fully functional cluster locally with binaries built from source.
+
+#### Updating the Node binaries
+
+If you already have an existing cluster you also test your changes by swapping the binaries on a node.
+To update the binaries on an existing node, follow these steps:
+
+1. Drain & cordon a node with `kubectl drain <nodename>`.
 2. Connect to the node with SSH or Windows Remote Desktop, and start PowerShell
-3. On the node, run `Stop-Service kubelet -Force`
+3. On the node, run `Stop-Service kubelet -Force`.
 4. Copy kubelet.exe and kube-proxy.exe to a cloud storage account, or use SSH to copy them directly to the node.
```

Comment:
I suggest saying `scp the binaries directly to the node`. Mentioning a cloud storage account might muddy the waters.

Reply:
updated